tables/nl-NL-g0.utb.in

```diff
@@ -0,0 +1,367 @@
+nofor  lowercase ε 15          GREEK LETTER EPSILON
+noback lowercase ζ 1356b       GREEK LETTER ZETA
+nofor  lowercase ζ 1356        GREEK LETTER ZETA
+noback lowercase ι 24b         GREEK LETTER IOTA
+nofor  lowercase ι 24          GREEK LETTER IOTA
+noback lowercase ί 24c         GREEK LETTER IOTA WITH TONOS
+nofor  lowercase ί 24          GREEK LETTER IOTA WITH TONOS
+noback lowercase κ 13b         GREEK LETTER KAPPA
+nofor  lowercase κ 13          GREEK LETTER KAPPA
+noback lowercase λ 123b        GREEK LETTER LAMDA
+nofor  lowercase λ 123         GREEK LETTER LAMDA
+noback lowercase μ 134b        GREEK LETTER MU
+nofor  lowercase μ 134         GREEK LETTER MU
+noback lowercase ν 1345b       GREEK LETTER NU
+nofor  lowercase ν 1345        GREEK LETTER NU
+noback lowercase ξ 1346b       GREEK LETTER XI
+nofor  lowercase ξ 1346        GREEK LETTER XI
+noback lowercase ο 135b        GREEK LETTER OMICRON
+nofor  lowercase ο 135         GREEK LETTER OMICRON
+noback lowercase π 1234b       GREEK LETTER PI
+nofor  lowercase π 1234        GREEK LETTER PI
+noback lowercase ρ 1235b       GREEK LETTER RHO
+nofor  lowercase ρ 1235        GREEK LETTER RHO
+noback lowercase σ 234b        GREEK LETTER SIGMA
+nofor  lowercase σ 234         GREEK LETTER SIGMA
+noback lowercase ς 234c        GREEK LETTER FINAL SIGMA
+nofor  lowercase ς 234         GREEK LETTER FINAL SIGMA
+noback lowercase τ 2345b       GREEK LETTER TAU
+nofor  lowercase τ 2345        GREEK LETTER TAU
+noback lowercase υ 136b        GREEK LETTER UPSILON
+nofor  lowercase υ 136         GREEK LETTER UPSILON
+noback lowercase φ 124b        GREEK LETTER PHI
+nofor  lowercase φ 124         GREEK LETTER PHI
+noback lowercase ϕ 124c        GREEK PHI SYMBOL
+nofor  lowercase ϕ 124         GREEK PHI SYMBOL
+noback lowercase ψ 13456b      GREEK LETTER PSI
+nofor  lowercase ψ 13456       GREEK LETTER PSI
+noback lowercase ω 2456b       GREEK LETTER OMEGA
+nofor  lowercase ω 2456        GREEK LETTER OMEGA
+noback lowercase η 156b        GREEK LETTER ETA
+nofor  lowercase η 156         GREEK LETTER ETA
+noback lowercase θ 1456b       GREEK LETTER THETA
+nofor  lowercase θ 1456        GREEK LETTER THETA
+noback lowercase χ 12346b      GREEK LETTER CHI
+nofor  lowercase χ 12346       GREEK LETTER CHI
+
+base uppercase Α α
+base uppercase Ά ά
+base uppercase Β β
+base uppercase Γ γ
+base uppercase Δ δ
+base uppercase Ε ε
+base uppercase Ζ ζ
+base uppercase Ι ι
+base uppercase Ί ί
+base uppercase Κ κ
+base uppercase Λ λ
+base uppercase Μ μ
+base uppercase Ν ν
+base uppercase Ξ ξ
+base uppercase Ο ο
+base uppercase Π π
+base uppercase Ρ ρ
+base uppercase Σ σ
+base uppercase Τ τ
+base uppercase Υ υ
+base uppercase Φ φ
+base uppercase Ψ ψ
+base uppercase Ω ω
+base uppercase Η η
+base uppercase Θ θ
+base uppercase Χ χ
+
+attribute foreign ΑαΆάΒβΓγΔδΕεΖζΙιΊίΚκΛλΜμΝνΞξΟοΠπΡρΣσςΤτΥυΦφϕΨψΩωΗηΘθΧχ
+
+# Other alphabets
+
+attribute foreign Ññ # ñ defined in nl-chardefs.uti
+
+_foreignuplow(ł,Ł,126)
```

Comment:
One the one hand I like my invocation syntax better, because it looks like a regular Liblouis rule. Of course on the other hand, it will be clearer, especially for newcomers who are less familiar with the available built-in opcodes, if macro invocations have a different syntax.

I think it would be ideal if the macro would start with an underscore, but if the parentheses were gone.

Reply:
> I think it would be ideal if the macro would start with an underscore, but if the parentheses were gone.

I explicitly defined the macro so it does start with an underscore. Mostly so that it would not clash with any other string in the table (that would then also get replaced by `m4`)

As for the parentheses, I looked through the documentation and that doesn't seem to be possible. The only way you can invoke a macro with no parentheses is when the macro has no arguments.